.github/CHANGELOG.md

```diff
@@ -231,6 +289,30 @@
   quantum tapes and classical processing functions, evaluate all quantum tapes simultaneously,
   and then apply the post-processing functions to the evaluated tape results.
 
+* The MultiRZ gate now has a defined generator, allowing it to be used in quantum natural gradient
+  optimization.
+  [(#912)](https://github.com/PennyLaneAI/pennylane/pull/912)
+
+* The CRot gate now has a `decomposition` method, which breaks the gate down into rotations
+  and CNOT gates. This allows `CRot` to be used on devices that do not natively support it.
+  [(#908)](https://github.com/PennyLaneAI/pennylane/pull/908)
+
+* The classical processing in the `MottonenStatePreparation` template has been largely
+  rewritten to use dense matrices and tensor manipulations where ever possible.
+  This is a preparation to support differentiation through the template in future.
+  [(#864)](https://github.com/PennyLaneAI/pennylane/pull/864)
+
+* Device-based caching has replaced QNode caching. Caching is now accessed by passing a
+  `cache` argument to the device.
+  [(#851)](https://github.com/PennyLaneAI/pennylane/pull/851)
+
+  The `cache` argument should be an integer specifying the size of the cache. For example, a
+  cache of size 10 is created using:
+
+  ```pycon
+  >>> dev = qml.device("default.qubit", wires=2, cache=10)
+  ```
+
 * The `Operation`, `Tensor`, and `MeasurementProcess` classes now has the `__copy__` special method
   defined. This allows us to ensure that, when a shallow copy is performed of an operation, the
   mutable list storing the operation parameters is *also* shallow copied. Both the old operation and
```

Comment:
Can't suggest, but just below, would be tood to move the PR link elsewhere. Looks a bit awkward to have a comma, then a bracket, then a code block

Reply:
done